internal/provider/provider.go

```diff
@@ -237,3 +259,40 @@ func detectDefaultConfig() (string, error) {
 
 	return "", errors.New("no mondoo config found")
 }
+
+type wif struct {
```

Comment:
Don't we have something like this already in cnquery and can we reuse it here?

Reply:
We do not. I looked and the implementation relies on `Viper` (https://github.com/spf13/viper) which is CLI specific.

I would like to consolidate this code in a follow up PR and decouple the use of viper/struct/token-exchange.